pset-functions.js

```diff
@@ -136,6 +178,46 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// Array scores of alphabet
+const letters = {
+    1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+    2: ['D', 'G'],
+    3: ['B', 'C', 'M', 'P'],
+    4: ['F', 'H', 'V', 'W', 'Y'],
+    5: ['K'],
+    8: ['J', 'X'],
+    10: ['Q', 'Z']
+}
+// Variable to count score
+let sum;
+
+// Function: return score of letter
+function lettersToScore(letter) {
+    for (let index in letters) {
+	if (Object.keys(letters).indexOf((letters[index].indexOf(letter) !== -1) ? index.toString() : '-1')
+	    !== -1) {
+	    return parseInt(index)
+	}
+    }
+}
+
+// Function: count final score
+const scrabbleScore = function(input) {
+    sum = 0
+    if (input === null || input === "") {
+	return 0
+    }
+
+    input.toUpperCase().split("").forEach(function(elem, index, array) {
+	sum += lettersToScore(elem)
+    })
+    return sum
+}
+
+console.log(scrabbleScore('cabbage'))
+console.log(scrabbleScore('javascript'))
+console.log(scrabbleScore('function'))
```

Comment:
@rlou88 Good work here on this solution. You are returning the correct value for each word and a good job creating an object with letters and points you get for scoring. One suggestion, for your, if statement on line 197, you may want to take.`Object.keys(letters).indexOf((letters[index].indexOf(letter) !== -1) ? index.toString() : '-1'` out the if steatment and store this expression in that variable